app/update/launcher.py

```diff
@@ -33,5 +33,6 @@ def start_async():
 
     update.result_store.clear()
 
-    subprocess.Popen(
-        ('sudo', '/usr/sbin/service', 'tinypilot-updater', 'start'))
+    with subprocess.Popen(
+        ('sudo', '/usr/sbin/service', 'tinypilot-updater', 'start')) as proc:
+        proc.wait()
```

Comment:
I believe adding `proc.wait()` here would cause `start_async` to stop being asynchronous, as it would wait for the child process to complete before the function returns.

I think we should suppress the warning here, too with a note like:

```python
# We don't need a context handler, as we're not managing the child process.
# pylint: disable=consider-using-with
```